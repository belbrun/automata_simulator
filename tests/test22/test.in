b,a
q0,q1,q2
b,a,c
A,K
q2
q0
K
q0,b,K->q0,AK
q0,b,A->q0,AA
q0,a,A->q1,A
q1,c,A->q1,$
q1,$,K->q2,$
