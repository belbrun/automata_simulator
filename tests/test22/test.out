q0#K|q0#AK|q1#AK|0
